File: app/src/main/java/org/mokee/warpshare/ItemPeerViewHolder.java
package org.mokee.warpshare;

import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import org.mokee.warpshare.databinding.ItemPeerMainBinding;

public class ItemPeerViewHolder extends RecyclerView.ViewHolder {
    ItemPeerMainBinding mBinding;
    public ItemPeerViewHolder(@NonNull View itemView) {
        super(itemView);
        mBinding = ItemPeerMainBinding.bind(itemView);
    }
}

File: app/src/main/java/org/mokee/warpshare/ui/ItemPeerViewHolder.kt
package org.mokee.warpshare.ui

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import org.mokee.warpshare.databinding.ItemPeerBinding

class ItemPeerViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var mBinding: ItemPeerBinding

    init {
        mBinding = ItemPeerBinding.bind(itemView)
    }
}